src/main/kotlin/com/ajaxproject/warehouse/controller/CustomersController.kt

```diff
@@ -19,26 +19,30 @@ import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/customers")
-class CustomersController(val customerService: CustomerService) {
+class CustomersController(
+    val customerService: CustomerService
+) {
 
     @GetMapping
-    fun findAllCustomers(): List<CustomerDataLiteDto> = customerService.findAllCustomers()
+    fun findAllMongo(): List<CustomerDataLiteDto> = customerService.findAllCustomers()
```

Comment:
now your controller knows about databases, welcome to high coupling (even if it's just name) :)